### PR TITLE
fix: docs: address typo

### PR DESCRIPTION
**Pull request by nnsW3**
Hello
I found several spelling issues in your docs.
Hope it helps.

### CODE REVIEW

File: documentation/en/block-validation.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ domain separation tag.
 Draw randomness for current epoch with `WinningPoSt` domain separation tag.
 Get list of sectors challenged in this epoch for this miner, based on the randomness drawn. 
 
-`V`: Use filecoin proofs system to verify that miner proved access to sealed versions of these sectors.
+`V`: Use filecoin proofs system to verify that miner provided access to sealed versions of these sectors.
```
Comment (rvagg) [inline documentation/en/block-validation.md:109]:
```suggestion
`V`: Use filecoin proofs system to verify that miner proved access to sealed versions of these sectors.
```

this was correct

File: documentation/en/architecture/mpool.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ Use `MpoolPushMessage` instead.
 ### MpoolSub
 
 Returns a channel to receive notifications about updates to the message pool.
-Note that the context *must* be cancelled when the caller is done with the subscription.
+Note that the context *must* be canceled when the caller is done with the subscription.
```
Comment (rvagg) [inline documentation/en/architecture/mpool.md:55]:
```suggestion
Note that the context *must* be cancelled when the caller is done with the subscription.
```

this was fine as it was, it's the preferred spelling outside of the US but also valid in the US

**Comment by rvagg** [comment]
thanks for the contribution, it's just the one change that's relevant here, please adjust

**Comment by nnsW3** [comment]
done sir

**Review by rvagg — APPROVED** [review]
thanks

**Pull request merged**